internal/backends/mcu/lib.rs

```diff
@@ -44,6 +44,8 @@ pub trait Devices {
     fn time(&self) -> core::time::Duration {
         core::time::Duration::ZERO
     }
+    fn set_timer_interrupt(&mut self, _duration: core::time::Duration) {}
+    fn sleep(&self) {}
```

Comment:
I suggest to call this sleep_until_interrupt() or wait_for_interrupt() (even if it's not using wfe instead of wfi). Otherwise the sleep() functions typically take a timeout.

Reply:
Maybe these two function can be combined.  wait_interrupt_or_timeout